### DISCUSSION TITLE
Board name

### DISCUSSION THREAD

**Discussion by marmeladapk** [sinara-hw/EEM_FMC_Carrier#1] [General]
What should we call this board? I'd prefer a name corresponding to its function, but if we don't come up with anything I can look for some polish lake names that shouldn't be too hard to pronounce ;)

I wouldn't leave its name as Carrier or FMC Carrier, because that's super generic and will cause much confusion with other FMC carriers of the world.

**Comment by dhslichter** [comment]
SinaraFMC?

**Comment by kaolpr** [comment]
SinaraFMC may be misleading as it may same well be carrier and FMC.

**Comment by dhslichter** [comment]
"Sinara FMC Carrier", then?  Or is that too long?

**Comment by gkasprow** [comment]
What about using the name of some famous aircraft carrier? :)

**Comment by marmeladapk** [comment]
Hell no.

czw., 26 sie 2021, 10:52 użytkownik Greg Kasprowicz <
***@***.***> napisał:

**Comment by gkasprow** [comment]
Maybe one of these?
https://www.livescience.com/59073-10-animal-mothers-that-carry-babies-on-their-backs.html

**Comment by dhslichter** [comment]
"Backpacker"?
